### ISSUE TITLE
[0.9.4.2]influxd panic in backup operation

### ISSUE THREAD

**Issue by ZhangZhenhua** [influxdb/influxdb#4814]
influxd will throw out run time error in backup operation. And it's reproducible every time in my env. Here are my steps:
1. Following https://influxdb.com/docs/v0.9/administration/backup_and_restore.html, I first add below section to influxd configuration file to enable snapshot feature.
```
[snapshot]
   enabled = true 
```
2. run below command and it will panic. Had to restart the service.
```
root@influxdb001:~# /opt/influxdb/influxd backup /mnt/influxdb_bkp/influxdb.frist_bkp
2015/11/16 20:43:08 influxdb backup
2015/11/16 20:43:08 backup complete

```

Here is my basic env:
os: ubuntu 14.04.03
influxdb: 0.9.4.2

Please check the attached log file when it occurs.
[influxd.backup.panic.log.txt](https://github.com/influxdb/influxdb/files/36171/influxd.backup.panic.log.txt)

**Comment by otoolep** [comment]
I suggest you try 0.9.5-rc3. We fixed a few panics related to backup.

https://influxdb.com/download/index.html#

**Comment by beckettsean** [comment]
closing as duplicate of https://github.com/influxdb/influxdb/issues/3700

**Comment by ZhangZhenhua** [comment]
Thanks very much! Just to confirm that it got fixed in 0.9.5 rc3.
